airbyte-webapp/src/hooks/services/useWorkspace.tsx

```diff
@@ -29,11 +30,10 @@ const useWorkspace = () => {
   const analyticsService = useAnalyticsService();
 
   const finishOnboarding = async (skipStep?: string) => {
-    if (skipStep) {
-      analyticsService.track("Skip Onboarding", {
-        step: skipStep,
-      });
-    }
+    analyticsService.track(Namespace.ONBOARDING, Action.SKIP, {
```

Comment:
As I see previously this action was performed conditionally(is skipStep provided)
Was the condition removed intentionally?